modules/distr-tracing-document-attributes.adoc

```diff
@@ -34,6 +45,6 @@
 // Using the pattern ending in 'BookName' makes it easy to grep for occurrences
 // throughout the topics
 //
-:RN_BookName: Red Hat OpenShift distributed tracing Release Notes
+:RN_BookName: Red Hat OpenShift Distributed Tracing Release Notes
```

Comment:
I'm not sure if this is intentional to uppercase here when the rest of the PR uses lowercase, but perhaps it's intentional for anticipated grepping. Just pointing it out.

Reply:
Not even sure how that got in there, good catch!